src/SMG/ManagerBundle/Controller/ManagersController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace SMG\ManagerBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use FOS\RestBundle\Controller\Annotations;
+use SMG\UserBundle\Entity\User;
+use FOS\RestBundle\View\View;
+use FOS\RestBundle\Controller\FOSRestController;
+use Symfony\Component\HttpFoundation\Response;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+
+class ManagersController extends FOSRestController
+{
+    use Traits\TokenFromHeaderTrait;
+    use Traits\HandleUserTrait;
+
+    const TOKEN_DIGITS = 6;
+
+    /**
+     * @Annotations\Post("/user/add")
+     *
+     * @ParamConverter("user", converter="fos_rest.request_body")
+     */
+    public function createUserAction(User $user)
+    {
+        $this->throwIfClientNot('backend');
+        $manager = $this->get('fos_user.user_manager');
+
+        $token = $manager->deleteIfNonEnabledExists($user);
+
+        $errors = $this->validates($user);
+        if (count($errors) > 0) {
+            return $this->handleView(
+                new View($errors, Response::HTTP_BAD_REQUEST)
+            );
+        }
+
+        $newUser = $manager->createUser();
+
+        $phoneNumber = $user->getPhoneNumber();
+        $email = $user->getEmail();
+
+        if (!is_null($phoneNumber)) {
+            $phoneNumber = str_replace('+', '00', $phoneNumber);
+        }
+
+        // TODO: refactor common parameters 
+        // settings in the HandleUserTrait.
```

Comment:
even before , starting from the    +        $token = $manager->deleteIfNonEnabledExists($user);   (not that here token is not used)